content/client-server-api/_index.md

```diff
@@ -208,8 +222,6 @@ Some API endpoints may allow or require the use of `POST` requests
 without a transaction ID. Where this is optional, the use of a `PUT`
 request is strongly recommended.
 
-{{% http-api spec="client-server" api="versions" %}}
```

Comment:
and moved this into "discovery" (for want of a better place), for the same reason.